helpers/admin/adminHelpers.ts

```diff
@@ -58,6 +58,10 @@ export const makeGraphqlVariable = (options: any, addProp?: any) => {
     res.id = parseInt(res.id ? res.id + '' : '')
   }
 
+  if (res.hasOwnProperty('lessonId')) {
```

Comment:
Question: How does `challenges` work? DOesn't `challenge` also have a field for `lessonId`?

Reply:
It doesn't ask for the lesson. When you click a lesson all the challenges currently come with it. When you add a challenge you are on the lesson page already.